Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -104,6 +104,7 @@
   - MedkitAdvanced
   - MedkitRadiation
   - MedkitCombat
+  - felinidcone
```

Comment:
Casing

Reply:
You also need to make a lathe prototype with its cost, etc for this to work. This isn't looking for the entity proto, it's looking for the lathe recipe proto so it knows how many materials to charge, etc